test/functional/api/window_spec.lua

```diff
@@ -860,6 +839,7 @@ describe('API/win', function()
   end)
 
   describe('text_height', function()
+    local text_height = api.nvim_win_text_height
```

Comment:
nit: aliasing the api functions in tests makes it hard to grep for existing test cases. and doesn't look like it gains much here.